crates/apollo-compiler/test_data/serializer/diagnostics/0125_query_shorthand_description.graphql

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
because this is in `diagnostics`, i am assuming you wanted to test the shorthand with a description, which is not valid with the new spec. i think this is missing that description 😅

Reply:
That's expected: this is the output of reserializing the document, and the serializer should not output invalid GraphQL (so it omits the description)